server/routes/static.py

```diff
@@ -14,21 +14,22 @@
 """Data Commons static content routes."""
 
 from datetime import date
-from flask import Blueprint, render_template, current_app, g
-from lib.gcs import list_blobs
+from flask import Blueprint, render_template, g, current_app
 import babel.dates as babel_dates
 
-_SA_FEED_BUCKET = 'datacommons-frog-feed'
-_MAX_BLOBS = 1
-
 bp = Blueprint('static', __name__)
 
 
 @bp.route('/')
 def homepage():
+    # TODO: Update this to use ENV_NAME
     if current_app.config.get('PRIVATE', None):
-        return render_template('static/private.html')
+        return render_template('private_dc/default/homepage.html')
+    if current_app.config.get('FEEDING_AMERICA', None):
+        return render_template('private_dc/feeding_america/homepage.html')
```

Comment:
Same comment as an earlier one: we should use "feedingamerica" as one word now :)